src/views/AccountDetail.vue

```diff
@@ -3,9 +3,8 @@
         <div class="page-content-card-f">
 
 
-            <Card 
+            <Card
                 class="card-f card-full-width"
-                :loading="loading"
```

Comment:
I think we need loading animation here and you?

Reply:
i was missing loading in computed().